main.py

```diff
@@ -0,0 +1,21 @@
+from fastapi import FastAPI
+import json
+
+app = FastAPI()
+
+with open('events.json', 'r') as file:
+    events = file.read()
+
+all_events = json.loads(events)
+
+
+@app.get('/get_event/{customer_id}')
```

Comment:
This works for now, but it does provide an opportunity to mention REST apis which is a widely used approach to building API's. I suggest you study some documentation or the original paper.

With REST conventions this endpoint's path should point to the events resource, thus it could be `/events/{customer_id}`